### DISCUSSION TITLE
Recommendations for hiking

### DISCUSSION THREAD

**Discussion by gill876** [NAIST-SE/AttractingContributors#1089] [2024 Attraction Contributions]
# ~~Hey there!~~ Thank you!

~~Do you like hiking? Got any places you'd recommend?~~

~~Please tell me!~~

Thank you for your contributions! I already have enough PRs so I closed this discussion.
If you'd like for me to contribute to your project, you can mention me!

## Project link:
https://github.com/gill876/AttractingContributors-Hiking

## How to make Pull Request to my repository

Go to the directory where you have your forked NAIST-SE project via terminal, and do the following!

`$ git remote add YOUR-HOBBY https://github.com/gill876/AttractingContributors-Hiking`
`$ git fetch YOUR-HOBBY main`
`$ git checkout -b YOUR-HOBBY-YOURNAME YOUR-HOBBY/main`

_ADD YOUR HOBBY_

`$ git add README.md`
`$ git commit -m "YOUR COMMIT MESSAGE"`
`$ git push origin YOUR-HOBBY-YOURNAME`

Go to [my project](https://github.com/gill876/AttractingContributors-Hiking) and create a pull request!
Done!

Thanks in advance!

Thanks @takuma1229 for the [ingenious instruction format!](https://github.com/NAIST-SE/AttractingContributors/discussions/1043#discussion-6833341)!

**Comment by hy20191108** [comment]
I guess Mt. Nijo.

**Reply by gill876** [reply]
Thank you so much for your suggestion!
Please consider contributing to my project by making
a PR with your suggestion in my README.md! 🙏🏿🙇🏿

https://github.com/gill876/AttractingContributors-Hiking

**Reply by hy20191108** [reply]
ok
Well, I think I've already sent out a pull request.
In the meantime, I have submitted my second pull request.

**Comment by buccalcone** [comment]
I add Mt.Ikoma.

If you can please contribute to my project!
https://github.com/YU762/AttractingContributors-Animals

**Comment by Canppchk** [comment]
I already contribute to your project. Thanks!!

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
